From 4f61a5b7ee5f14e14523b05c31e1de8673f72568 Mon Sep 17 00:00:00 2001
From: Xiyue Deng <manphiz@gmail.com>
Date: Sun, 29 Sep 2024 04:06:25 -0700
Subject: [PATCH] Move packages in Depends to Recommends

* This gives some flexibility for users that don't want certain
packages in the list.
* This may also lower the threshold of introducing new packages to
goodies as users have more flexibility to choose to install or not.
* Update the package description accordingly.
---
 debian/control | 41 +++++++++++++++++++++--------------------
 1 file changed, 21 insertions(+), 20 deletions(-)

diff --git a/debian/control b/debian/control
index 1d3f97f..2a45f03 100644
--- a/debian/control
+++ b/debian/control
@@ -14,29 +14,30 @@ Vcs-Git: https://salsa.debian.org/emacsen-team/emacs-goodies-el.git
 Package: emacs-goodies-el
 Architecture: all
 Depends: ${misc:Depends}
- , elpa-browse-kill-ring
- , elpa-diminish
- , elpa-htmlize
- , elpa-markdown-mode
- , elpa-csv-mode
- , elpa-graphviz-dot-mode
- , elpa-apache-mode
- , elpa-bm
- , elpa-boxquote
- , elpa-session
- , elpa-pod-mode
- , elpa-tabbar
- , elpa-eproject
- , elpa-initsplit
- , elpa-bar-cursor
- , elpa-mutt-alias
- , elpa-folding
- , elpa-muttrc-mode
-Recommends: elpa-color-theme-modern
+Recommends:
+ elpa-browse-kill-ring,
+ elpa-diminish,
+ elpa-htmlize,
+ elpa-markdown-mode,
+ elpa-csv-mode,
+ elpa-graphviz-dot-mode,
+ elpa-apache-mode,
+ elpa-bm,
+ elpa-boxquote,
+ elpa-session,
+ elpa-pod-mode,
+ elpa-tabbar,
+ elpa-eproject,
+ elpa-initsplit,
+ elpa-bar-cursor,
+ elpa-mutt-alias,
+ elpa-folding,
+ elpa-muttrc-mode,
+ elpa-color-theme-modern,
 Replaces: emacs-goodies-extra-el
 Provides: emacs-goodies-extra-el
 Description: Miscellaneous add-ons for Emacs
- This is a now a metapackage that installs the elpa-* packages
+ This is a now a metapackage that recommends the elpa-* packages
  corresponding to some of the previously bundled add-ons.
  .
  See /usr/share/doc/emacs-goodies-el/README.Debian.gz for a discussion
-- 
2.45.2

